canal/sync.go

```diff
@@ -141,12 +142,13 @@ func (c *Canal) handleRowsEvent(e *replication.BinlogEvent) error {
 	ev := e.Event.(*replication.RowsEvent)
 
 	// Caveat: table may be altered at runtime.
-	schema := string(ev.Table.Schema)
+	sch := string(ev.Table.Schema)
 	table := string(ev.Table.Table)
 
-	t, err := c.GetTable(schema, table)
+	t, err := c.GetTable(sch, table)
 	if err != nil {
-		return errors.Trace(err)
+		log.Errorf("GetTable info error %v", err)
+		return schema.LastGetTableInfoErr
```

Comment:
what does this error mean? 

You should define a no privilege error and check it, but not ignore any error and continue.